roles/mongodb/tasks/install.yml

```diff
@@ -91,7 +91,7 @@
       repo: deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/4.4 multiverse
       state: present
       filename: mongodb-org
-    when: is_ubuntu_20 and (ansible_architecture == "aarch64" or ansible_architecture == "amd64")
+    when: is_ubuntu_20
```

Comment:
Later we might test for Ubuntu > 19 (or similar) &mdash; so that things like is_ubuntu_21 (i.e. Ubuntu 21.04 and 21.10) have a better shot when 2021 creeps up...

Reply:
addressed in #2495